source/includes/_feed_items.md

```diff
@@ -26,13 +30,33 @@ $ http --form POST "https://api.getmondo.co.uk/feed" \
 
 Creates a new feed item on the user's feed
 
-##### Arguments
+##### General Arguments (for all types of feed item)
 
 <span class="hide">Parameter</span> | <span class="hide">Description</span>
 ------------------------------------|--------------------------------------
 `account_id`<br><span class="label notice">Required</span>|The account to create feed item for.
-`type`<br><span class="label notice">Required</span>|Type of feed item. Only supports `image` currently.
+`type`<br><span class="label notice">Required</span>|Type of feed item. Currently only `basic` is supported.
+`params`<br><span class="label notice">Required</span>|A *map* of parameters which vary based on `type`
+`url`<br><span class="label">Optional</span>|A URL to open when the feed item is tapped. If no URL is provided the feed item will expand on tap.
```

Comment:
Maybe "If no URL is provided, the app will display a fallback view based on the title & body"